charts/et-sya-api/values.preview.template.yaml

```diff
@@ -2,3 +2,11 @@ java:
   # Don't modify below here
   image: ${IMAGE_NAME}
   ingressHost: ${SERVICE_FQDN}
+  keyVaults:
+    et-cos:
+      secrets:
+        - name: et-cos-s2s-secret
+          alias: ET_COS_S2S_SECRET
```

Comment:
The service should have its own S2S secret

Reply:
Waiting for RET-913 to be completed, then we will have our own secret.